app/endpoint/create-room/ValidateCreateRoomRequest.ts

```diff
@@ -0,0 +1,50 @@
+import {validate} from '../../lib/validatejs/0_13_1/validate';
+
+export default function validateRequest(query: any): any {
+
+    console.log('query: ', query);
+
+    const constraints = {
+        'visitor': {
+            presence: {
+                allowEmpty: false,
+            },
+        },
+        'visitor.department': {
+            presence: {
+                allowEmpty: false,
+            },
+            type: 'string',
+        },
+        'visitor.token': {
+            presence: {
+                allowEmpty: false,
+            },
+            type: 'string',
+        },
+        'visitor.name': {
+            presence: {
+                allowEmpty: false,
+            },
+            type: 'string',
+        },
+        'visitor.priority': {
+            presence: false,
+            type: 'number',
+        },
+        'visitor.email': {
```

Comment:
Put a regex here too if possible.

Reply:
Validate.js lib already checks if the email is a valid one: https://validatejs.org/#validators-email